src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -1665,14 +1670,17 @@ internal static char SetDelimiter(PSCmdlet Cmdlet, string ParameterSetName, char
             switch (ParameterSetName)
             {
                 case "Delimiter":
+                case "DelimiterPath":
+                case "DelimiterLiteralPath":
                     //if delimiter is not given, it should take , as value
                     if (Delimiter == '\0')
                     {
                         Delimiter = ImportExportCSVHelper.CSVDelimiter;
                     }
 
                     break;
-                case "UseCulture":
+                case "CulturePath":
```

Comment:
If I understand this correctly, `"UseCulture"` should be added back in, because this method is used from the `Export-Csv` and `ConvertTo-Csv` cmdlets too, which still have the original parameter set names, `Delimiter` and `UseCulture`.

Reply:
Fixed.